test/common/verifier.go

```diff
@@ -23,8 +23,10 @@ import (
 
 const fileForAuditEvent = "/etc/newfile"
 
-// ExpectedAuditRules are the expected audit rules. Exposed so that they can be modified by tests.
-var ExpectedAuditRules = `## This file is automatically generated from /etc/audit/rules.d
+// ExpectedAuditRules is a set of expected audit rules to be deployed on the shoot nodes
+// when the AuditConfig in the shoot-rsyslog-relp extension providerConfig is enabled and
+// does not specify a ConfigMap reference containing custom audit rules.
+const ExpectedAuditRules = `## This file is automatically generated from /etc/audit/rules.d
```

Comment:
Nit: I would move the const to a private const in `test/testmachinery/shoot/enable_disable_test.go`. Right now, it is used only there. 

I would image something like:

```golang
const (
	defaultExtensionAuditRules = "..."
	customAuditRules = "..."
)
```